src/pages/Player.vue

```diff
@@ -44,14 +44,14 @@
 
 <script>
 import Plyr from "plyr";
-import PlioQuestion from "@/components/PlioQuestion.vue";
-import Error from "@/views/Error.vue";
-import LoadingSpinner from "@/components/LoadingSpinner.vue";
-import StartButtonPointer from "@/components/tutorial/StartButtonPointer.vue";
-import UserProperties from "@/components/UserProperties.vue";
-
-import PlioService from '@/services/PlioAPIService.js'
-import UserService from '@/services/UserAPIService.js'
+import PlioQuestion from "@/components/Items/PlioQuestion.vue";
+import Error from "@/pages/Error.vue";
+import LoadingSpinner from "@/components/UI/LoadingSpinner.vue";
+import StartButtonPointer from "@/components/UI/tutorial/StartButtonPointer.vue";
+import UserProperties from "@/utils/UserProperties.vue";
+
+import PlioService from '@/utils/APIservices/PlioAPIService.js'
```

Comment:
`API` can be a folder itself outside of `utils`

Reply:
API and all things related to it should not come under utils?

Reply:
Hmmm.. this one is up to you -- just giving my thoughts! 

For me, `utils` is a place for things that don't have any other place. ANything that is a subfolder inside `utils` means you know what it is and you can give it its own place.